pages/1.13/administering-clusters/locate-public-agent/index.md

```diff
@@ -6,23 +6,193 @@ menuWeight: 3
 excerpt: Finding a public agent IP address.
 enterprise: false
 ---
+After you have installed DC/OS with a public agent node declared, you can navigate to the public IP address of your public agent node. You can expose the public-facing IP address for an agent as a gateway for access to services that are running inside the DC/OS cluster. For example, if you are configuring load balancing to distribute inbound requests to the services in a cluster, the requests are typically routed through the public IP address frontend to an appropriate service instance backend isolated behind a firewall. 
 
-After you have installed DC/OS with a public agent node declared, you can navigate to the public IP address of your public agent node.
+# Before you begin
+- You must have DC/OS installed with at least one master node and at least one [public agent](/1.13/overview/concepts/#public-agent-node) node.
+- You must have the most recent version of the DC/OS [CLI](/1.13/cli/) installed.
+- You must have the [jq](https://github.com/stedolan/jq/wiki/Installation) language installed if you want to format the output from an API call.
```

Comment:
I've read too many RFC and `must` means its mandatory ;).

```suggestion
- You may use [jq](https://github.com/stedolan/jq/wiki/Installation) if you want to format the output from an API call.
```

Reply:
fixed.